docs/modular-resources/advanced/custom-components-remotes.md

```diff
@@ -8,51 +8,48 @@ aliases:
   - "/program/extend/sdk-as-server/"
   - "/program/extend/custom-components-remotes/"
   - "/extend/custom-components-remotes/"
-description: "Implement custom components and register them on a server configured as a remote of your robot."
+description: "If you are unable to use modular resources, you can implement custom components and register them on a server configured as a remote of your robot."
 ---
 
-{{% alert title="Tip" color="tip" %}}
+[Modular resources](/modular-resources/) running on the [board](/components/board/) directly connected to your components are the preferred way of managing and controlling custom components.
 
-[Modular resources](/modular-resources/) provided by custom {{< glossary_tooltip term_id="module" text="modules" >}} are the preferred method of creating custom resource implementations.
-
-This option is provided for advanced users who are unable to use the modular resource system with the robot deployments.
-
-{{% /alert %}}
-
-If a type or model of [component](/components/) you are working with is not [built-in to the Viam RDK](/internals/rdk/), or [available from the Viam registry as a module](/modular-resources/key-concepts/), you can use a [Viam SDK](/program/apis/) to code a custom resource implementation, host it on a server, and add it as a [remote](/manage/parts-and-remotes/) of your robot.
+However, if you are unable to use [modular resources](/modular-resources/) because you have to host `viam-server` on a non-Linux system or have an issue with compilation, you can use a [Viam SDK](/program/apis/) to code a custom resource implementation, host it on a server, and add it as a [remote](/manage/parts-and-remotes/) of your robot.
```

Comment:
```suggestion
However, if you are unable to use [modular resources](/modular-resources/) because you need to host `viam-server` on a non-Linux system or have an issue with compilation, you can use a [Viam SDK](/program/apis/) to code a custom resource implementation, host it on a server, and add it as a [remote](/manage/parts-and-remotes/) of your robot.
```
[opt]